hacks/isolateVersion/7-updateDocusaurusConfig.sh

```diff
@@ -0,0 +1,65 @@
+notify "Updating docusaurus.config.js..."
+
+# Remove unused versionedLinks dependency
+sed -i '' '/const versionedLinks/d' docusaurus.config.js
+
+# Update `url` to include `unsupported`
+sed -i '' -e "s/docs.camunda.io/unsupported.docs.camunda.io/" docusaurus.config.js
+
+# Update `baseUrl` to specify isolated version
+sed -i '' -e "s/baseUrl: \"\\/\"/baseUrl: \"\/$ARCHIVED_VERSION\/\"/" docusaurus.config.js
```

Comment:
These commands don't need the `-e`! That flag is used to give sed multiple commands, but these are broken into individual statements. 

I'm going to remove those flags. ChatGPT is a dumb robot that keeps putting them in for no good reason (and I a dumb human who doesn't catch it).

Reply:
(I know that that's super minor, but these commands are all so cryptic and confusing that I want to eliminate any unnecessary noise.)

Reply:
🙏 thank you! Future me and future you thank you too!